scripts/elotl/repositories

```diff
@@ -1,2 +1,2 @@
-https://nl.alpinelinux.org/alpine/v3.9/main
-https://nl.alpinelinux.org/alpine/v3.9/community
+https://nl.alpinelinux.org/alpine/v3.11/main
```

Comment:
If you want to move to 3.11, we should also update the commit we check out from alpine-make-vm-image: https://github.com/elotl/itzo/blob/master/scripts/build-image.sh#L157

Reply:
Good catch I have addressed that in the most recent commit. I added a checkout for the commit at HEAD of master from alpine-make-vm-image repo so we can stay stable.